src/python/zigopt/sigoptcompute/adapter.py

```diff
@@ -237,6 +238,21 @@ def multisolution_best_assignments(
     best_indices = [int(i) for i in response["best_indices"] if i is not None]
     return best_indices
 
+  def random_search_next_points(
+    self,
+    experiment,
+    num_to_suggest,
+    tag=None,
+  ):
+    view_input = {
+      "domain_info": self.generate_domain_info(experiment, only_active_categorical_values=True),
+      "num_to_sample": num_to_suggest,
+      "tag": self.supplement_tag_with_experiment_id(tag, experiment),
```

Comment:
If we aren't using the tag might as well pass in an empty dict

Reply:
the tag makes sense here for debugging